cuda_core/cuda/core/experimental/_linker.py

```diff
@@ -42,6 +43,12 @@ def _decide_nvjitlink_or_driver():
             _nvjitlink = None
 
     if _nvjitlink is None:
+        warnings.warn(
+            "nvJitLink is not installed or too old (<12.3). Therefore it is not usable "
+            "and the culink APIs will be used instead.",
+            stacklevel=3,
```

Comment:
Q: Is `stacklevel=2` not enough? I always forget which level is the right choice, but I'd prefer to not print out the cuda.core internal as part of the traceback.